src/Helpers/BlockRenderer.php

```diff
@@ -230,7 +230,10 @@ private static function getNestedBlocksForBlock(
         $class = Block::findFirstWithType($block->type);
 
         /** @var \A17\Twill\Models\Block[] $childBlocks */
-        $childBlocks = A17Block::whereParentId($block->id)->get();
+        $childBlocks = A17Block::whereParentId($block->id)->orderBy(
+            config('twill.blocks_table', 'twill_blocks') . '.position',
```

Comment:
Makes sense, however, I wonder, do we really need to recall the block table name? I feel like that's already assumed because of the A17Block model?

Reply:
Hey, yes please

Reply:
Hey @mikerockett the tests are indeed unrelated, will work on that today